src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -6210,6 +6210,75 @@ def test_sql_midb_move_copy(self, mi1, rg, mi2):
                                     self.check('[0].resourceGroup', '{rg}'),
                                     self.check('[0].operationMode', 'Copy')])
 
+    @live_only()
```

Comment:
You can commit the recording file and mark it as record only

Reply:
Okay, I will change it from live_only to record_only

Reply:
Could you please explain the difference between live_only() and record_only() ?

Reply:
All tests should be executed successfully in live mode and record mode.

But for some of them which we don't want to record (for example, some credentials may be recorded)/we are not able to record since time past and code changes, we mark them as `live_only`

And for some of them which CLI team can't run successfully due to fixed resource name or permission issue, we ask service team to record and then mark as `record_only`

Reply:
Well, the tests are failing in the pipeline. Is that because I put placeholders and the recording(.yml file) has real values?